src/localization/en.json

```diff
@@ -9,11 +9,39 @@
     "yes": "Yes",
     "no": "No",
     "loadingAutoCompleteOptions": "Loading options..",
-    "confirm": "Confirm"
+    "confirm": "Confirm",
+    "close": "Close",
+    "reload": "Reload"
   },
 
   "errorHandling": {
-    "title": "Error"
+    "title": "Error",
+    "dialog": {
+      "title": "Whoops! Something went wrong",
+      "reloadPage": "Try reloading the page.",
+      "unsavedContents": "Please note that unsaved content will not be saved when reloading. If you have unsaved content on this page, you can close this message and copy the content to the clipboard before reloading.",
+      "reportIssue": "If the problem persists, contact support@metatavu.fi by email. Please include technical details in to the email to speed up resolution of the problem.",
+      "technicalDetails": "technical details:",
```

Comment:
```suggestion
      "technicalDetails": "Technical details:",
```
I'd probably make the first letter uppercase as not consistent at the moment with the other lines.
![image](https://user-images.githubusercontent.com/5682352/181730213-2d9a0758-012d-4ead-8729-bf6d176ccece.png)